ci/cron/daily-compat.yml

```
@@ -116,3 +116,64 @@ jobs:
       - template: ../daily_tell_slack.yml
         parameters:
           success-message: '$(cat $(Build.StagingDirectory)/perf-results.json | jq . | jq -sR ''"perf for ''"$COMMIT_LINK"'':```\(.)```"'')'
+
+  - job: perf_http_json
+    timeoutInMinutes: 120
+    pool:
+      name: "linux-pool"
+      demands: assignment -equals default
+    steps:
+      - checkout: self
+      - bash: ci/dev-env-install.sh
+        displayName: 'Build/Install the Developer Environment'
+      - bash: ci/configure-bazel.sh
+        displayName: 'Configure Bazel for root workspace'
+        env:
+          IS_FORK: $(System.PullRequest.IsFork)
+          # to upload to the bazel cache
+          GOOGLE_APPLICATION_CREDENTIALS_CONTENT: $(GOOGLE_APPLICATION_CREDENTIALS_CONTENT)
+      - template: ../bash-lib.yml
+        parameters:
+          var_name: bash_lib
+      - bash: |
+          set -euo pipefail
+          eval "$(dev-env/bin/dade assist)"
+          source $(bash_lib)
+
+          SCENARIOS="\
+           com.daml.http.perf.scenario.CreateCommand \
+           com.daml.http.perf.scenario.ExerciseCommand \
+           com.daml.http.perf.scenario.CreateAndExerciseCommand \
+           com.daml.http.perf.scenario.AsyncQueryConstantAcs \
+           com.daml.http.perf.scenario.SyncQueryConstantAcs \
+           com.daml.http.perf.scenario.SyncQueryNewAcs \
+           com.daml.http.perf.scenario.SyncQueryVariableAcs \
+          "
+
+          DAR=/var/tmp/quickstart-model.dar
+
+          bazel build //docs:quickstart-model
+          # sandbox cannot open a dar via bazel symlink, copy it into /var/tmp
+          cp -vf ./bazel-bin/docs/quickstart-model.dar ${DAR}
+
+          JWT="eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJodHRwczovL2RhbWwuY29tL2xlZGdlci1hcGkiOnsibGVkZ2VySWQiOiJNeUxlZGdlciIsImFwcGxpY2F0aW9uSWQiOiJmb29iYXIiLCJhY3RBcyI6WyJBbGljZSJdfX0.VdDI96mw5hrfM5ZNxLyetSVwcD7XtLT4dIdHIOa9lcU"
+
+          START=$(date -u +%Y%m%d_%H%M%SZ)
```

How about changing this to something similar to our release format for snapshots (https://github.com/digital-asset/daml/blob/09014dc63729e4a126306ec22183ded68917eb40/release.sh#L58)? Date makes it hard to trace it back to a commit.

---

You only linked one line of it here, I suppose you'd want a bit more, like:
```suggestion
          START=$(git log -n1 --format=%cd --date=format:%Y%m%d).$(git rev-list --count HEAD).$(git log -n1 --format=%h --abbrev=8)
```

---

`job: perf_speedy` uses the same approach. So do you want to change it there too?